debian/default

```diff
@@ -0,0 +1,2 @@
+SNAPCAST_SERVEROPTS="-B 120 -b 25000 -s /tmp/snapfifo"
```

Comment:
-B is now obsolete. I suggest to alter the defaults to:
SNAPCAST_SERVEROPTS="-s "pipe:///tmp/snapfifo?name=default&codec=opus&sampleformat=48000:16:2&buffer_ms=120" -b 30000 -p 1704"
SNAPCAST_CLIENTOPTS="-s snapclient -m snapclient -H media.lan -p 1704"

Reply:
i used localhost now. media seems the name on your machine? ;) localhost seemed to me to be a sane default?